packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -38,6 +39,7 @@ function InterfaceSkeleton( {
 	const defaultLabels = {
 		header: __( 'Header' ),
 		body: __( 'Content' ),
+		leftSidebar: __( 'Left sidebar' ),
```

Comment:
Shouldn't we call this "Block Library" or "Insert Blocks"?

Reply:
I guess so, this is a generic UI component though, so I'll pass a labels prop.